terraform.tfvars

```diff
@@ -17,3 +19,9 @@ desired_capacity = 2
 instance_type = "t2.micro"
 
 ecs_aws_ami = "ami-c1a6bda2"
+
+owner = "alexander.vincent@ga.gov.au"
```

Comment:
Good work adding the owner tag, we've historically run Janitor Monkey to clean up resources that don't have the Owner tag, as it makes it much easier to debug issues if you know who built the thing.